docker-scripts/docker-compose-klaw-v2.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-version: '4'
```

Comment:
I got an error for an invalid version in the github pipeline and after reading it up, as far as I [understood it](https://github.com/compose-spec/compose-spec/blob/master/04-version-and-name.md) the `version` field is no longer used and only informative, so I removed it - but I can add it back in again. Although I think there is no version 4 for docker-compose anyways? So not sure what this is for.

Reply:
I believe I was miss-using this to set the version of the docker-compose file it's self, basically just informing as well which version of the file it is.